common/spring-cloud-stream-modules-common-configuration/src/main/java/org/springframework/cloud/stream/module/trigger/TriggerConstants.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright 2015 the original author or authors.
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.cloud.stream.module.trigger;
+
+/**
+ * @author Ilayaperumal Gopinathan
+ */
+public class TriggerConstants {
+
+	public static final String TRIGGER_BEAN_NAME = "SPRING_CLOUD_STREAM_TRIGGER";
```

Comment:
I don't like this name.
I really can run any module outside of the cloud.
Why simple `pollerTrigger` name isn't enough for you?
Moreover this definition is in the `module` package...